### ISSUE TITLE
Bug: Сompiler crashes without error description

### ISSUE THREAD

**Issue by pyAndr3w** [everx-labs/TVM-Solidity-Compiler#137]
If you make a try-catch construct with a syntax error*, then the compiler crashes

> *is this a syntax error?

---

solidity code for testing:
```solidity
contract Test {
    function testFunc() private {
        try {
            // pass
        } catch {
            // pass
        }
    }
}
```

message in console:
```
Segmentation fault (core dumped)
```

---

compiler version: 0.69.0

**Comment by IgorKoval** [comment]
Thank you! We are going to fix it in v0.71.0

**Comment by cryshado** [comment]
Fixed in 0.71.0 (2023-07-20) #141
